packages/@sfx/sayt-plugin/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "@sfx/sayt-plugin",
+  "version": "0.0.0",
+  "description": "",
+  "main": "src/index.ts",
+  "scripts": {
+    "build": "../../../scripts/build.sh",
+    "dev": "nodemon --watch src --exec npm run build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Let's throw this in for reference as per https://docs.npmjs.com/files/package.json#repository
```suggestion
  },
  "repository": {
  "type" : "git",
  "url" : "https://github.com/groupby/sfx-logic.git",
  "directory": "packages/@sfx/sayt-plugin"
},
```

---

Updated.